pynxtools_xps/reader_utils.py

```diff
@@ -453,6 +453,7 @@ def align_name_part(name_part: str):
         ",": "",
         ".": "_",
         "-": "_",
+        ":": "_",
```

Comment:
Just as a suggestion: You cold also regex everything which is not an accepted nexus character and replace it with `_`. That would ensure that the names are always nexus safe (but could create weird combinations of `_`s). But also fine like this.

Reply:
I may eventually replace it by different characters, so I avoid weird combinations of `_`s). So, I'll just keep this for now.